public/apps/configuration/panels/permission-list.tsx

```diff
@@ -0,0 +1,242 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import {
+  EuiButton,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageHeader,
+  EuiPopover,
+  EuiText,
+  EuiTitle,
+  EuiIcon,
+} from '@elastic/eui';
+import { difference } from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { AppDependencies } from '../../types';
+import { ActionGroupListingItem, fetchActionGroupListing } from '../utils/action-groups-utils';
+import { renderCustomization } from '../utils/display-utils';
+import { requestDeleteUsers } from '../utils/internal-user-list-utils';
+
+
+function renderBooleanToCheckMark(value: boolean): React.ReactNode {
+  return value ? <EuiIcon type="check" /> : ''
+}
+
+const COLUMNS = [
```

Comment:
Missing a column "Created". Will it be added in next iteration?

Reply:
Probably not, the backend does not store this information.